install_script.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+RELEASE_DIR=HCK_CI
+INSTALL_PATH=~/HCK-CI
+
+INSTALL_QEMU=yes
+INSTALL_GEMS=yes
+INSTALL_DHCP=yes
+GITHUB_REPO=
+GITHUB_LOGIN=
+GITHUB_TOKEN=
+DROPBOX_TOKEN=
+
+if [ -z "$GITHUB_REPO" ]; then
+  echo "WARNING: github repository value is empty"
+fi
+
+if [ -z "$GITHUB_LOGIN" ]; then
+  echo "WARNING: github login value is empty"
+fi
+
+if [ -z "$GITHUB_TOKEN" ]; then
+  echo "WARNING: github token value is empty"
+fi
+
+
+if [ -z "$DROPBOX_TOKEN" ]; then
+  echo "WARNING: Dropbox token value is empty"
+fi
+
+if [ -z "$INSTALL_PATH" ]; then
+  echo "Installation path Already exists."
+fi
+
+echo Copying files to installtion path
+mkdir -p $INSTALL_PATH
+cd $INSTALL_PATH
+#Get absolute path 
+INSTALL_PATH=$(pwd)
+cd -
+cp -rf $RELEASE_DIR/* $INSTALL_PATH
+
+if [[ "$INSTALL_QEMU" == "yes" ]]; then
+  cd $INSTALL_PATH
+  git clone https://github.com/qemu/qemu.git
+  cd qemu
+  ./configure --disable-docs --target-list=x86_64-softmmu
+  make -j
+  cd -
+  qemu=$INSTALL_PATH/x86_64-softmmu/qemu-system-x86_64
+  qemu_img=$INSTALL_PATH/qemu_img
+  ivshmem_server=$INSTALL_PATH/ivshmem_server
+fi
+
+if [[ "$INSTALL_GEMS" == "yes" ]]; then
+  cd $INSTALL_PATH/gems
+  gem install *
+  cd -
+fi
+
+if [[ "$INSTALL_DHCP" == "yes" ]]; then
+  cd $INSTALL_PATH/OpenDHCPServerSetup
+  sudo ./opendhcpserverSetup.sh
+  cd -
+fi
+
+cat > $INSTALL_PATH/AutoHCK/config.json <<EOF
```

Comment:
Print path to json and say that it can be updated.
Print other config json files as well